scripts/submit_bb.py

```diff
@@ -12,6 +14,28 @@
 default_wct = "00:30:00"
 
 
+def generate_context(
```

Comment:
Instead of creating a new generate_context (and generate_header) function for every submit script, create a single shared function. As these functions all do the same thing, the only difference are the arguments, use **kwargs to deal with that. 

If you are not sure what I mean, let me know, I'm happy to explain.

Reply:
Can we still use kwargs when setting the function_parameter name? I wasn't sure about that otherwise I would have suggested it.